testbed/tests/widgets/test_table.py

```diff
@@ -98,6 +98,18 @@ async def multiselect_widget(source, on_select_handler):
     )
 
 
+@pytest.fixture
```

Comment:
If the widget and probe is identical, we don't need to repeat it. 

In general, we only use skip in the testbed itself as a marker that the widget isn't implemented at all, not as an indicator of test availability.

It looks like the issue is the lack of shift handling on macOS; that's easy for me to add; I'll push an update shortly that adds that feature to the macOS testbed.